pkg/codegen/python/gen.go

```diff
@@ -460,13 +485,30 @@ func (mod *modContext) importFromToken(tok string, input bool) string {
 	return fmt.Sprintf("from %s import %[2]s as _%[2]s", relImport, components[0])
 }
 
-// emitConfigVariables emits all config vaiables in the given module, returning the resulting file.
+func (mod *modContext) importResourceFromToken(tok string) string {
```

Comment:
Now that we have `importResourceFromToken`, should we rename `importFromToken` to `importTypeFromToken`?

Reply:
Sounds reasonable